### PR TITLE
Feature/small bug fixes

### PR DESCRIPTION
**Pull request by cblignaut**
Hi @saeedmb8: If you have time could you please check this PR ?

### CODE REVIEW

File: molo/yourtips/admin.py [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
+download_as_csv.short_description = "Download selected as csv"
 
 
 @staff_member_required
 def convert_to_article(request, entry_id):
+    """
+    This method converts a tip entry to an unpublished article
+    :param entry_id: Tip entry to convert
+    :return: redirect to the edit page of the converted tip
+    """
```
Comment (smarzban) [inline molo/yourtips/admin.py:40]: 😍

File: molo/yourtips/models.py [hunk 2]
```diff
@@ -60,6 +66,15 @@ def create_yourtips_section_index_page(sender, instance, **kwargs):
...
 class YourTip(TranslatablePageMixinNotRoutable, Page):
+    """
+    This model links the views and store settings for the yourtips module.
+    Note that:
+    * title: Is used throughout the module where the name of
+        the module need to be displayed.
+    * description: Is used to display the YourTip description.
+    * homepage_action_copy: Is used to change the text of the banner.
+    * extra_style_hints: Is used to change style of the link on the main page.
+    """
```
Comment (smarzban) [inline molo/yourtips/models.py:77]: 😍

**Comment by smarzban** [comment]
👍 🍰 👌🏼

**Comment by cblignaut** [comment]
Yay :)

**Pull request merged**